content/blog/2010/2010-07-28-hosting-your-hudson-plugin-at-github.html

```diff
@@ -60,7 +60,7 @@
 <br/>
 As with the first section, this is needed to override the default SCM settings in the parent POM. Make sure to change "your-plugin-repo" to your actual plugin repository name, of course! It'd also be a good idea to put a link to your Github repo on your plugin's wiki page at <a href="https://wiki.jenkins.io">the Hudson wiki</a>, since the default source link will go to Subversion.<br/>
 <br/>
-Now, with these changes in place, committed and pushed to the master branch in your plugin's repository, you should be able to run "mvn release:prepare release:perform" just as you would if the plugin were hosted in Subversion, with your plugin showing up in the Update Center within a few hours. Be sure that you've configured the login information for the java.net Maven repository, as detailed <a href="https://javanettasks.dev.java.net/nonav/maven/config.html">here</a> - this is still needed for plugins hosted at and released from Github, since Hudson's Maven repository still lives at java.net. If you run into any issues releasing your plugin from Github, first be sure to review the <a href="https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins">wiki page on plugin hosting</a>, which addresses many of the issues you may encounter when running the Maven release plugin. <br/>
+Now, with these changes in place, committed and pushed to the controller branch in your plugin's repository, you should be able to run "mvn release:prepare release:perform" just as you would if the plugin were hosted in Subversion, with your plugin showing up in the Update Center within a few hours. Be sure that you've configured the login information for the java.net Maven repository, as detailed <a href="https://javanettasks.dev.java.net/nonav/maven/config.html">here</a> - this is still needed for plugins hosted at and released from Github, since Hudson's Maven repository still lives at java.net. If you run into any issues releasing your plugin from Github, first be sure to review the <a href="https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins">wiki page on plugin hosting</a>, which addresses many of the issues you may encounter when running the Maven release plugin. <br/>
```

Comment:
Another case where this is referring to the "master" branch, not the Jenkins "master"